_includes/accessibility-tests/a11y-note.html

```diff
@@ -1,13 +1,15 @@
+{% assign component_name = page.title | downcase %}
+
 <div class="usa-alert usa-alert--info usa-alert--no-icon measure-5 margin-bottom-2">
   <div class="usa-alert__body">
     <p class="usa-alert__text text-bold">
-      Test the {{ componentName }} in your own project.
+      Test the {{ component_name }} component in your own project.
     </p>
     <p class="margin-y-1">
-      USWDS tested the {{ componentName }}  component for accessibility. You should test your implementation, too.
+      USWDS tested the {{ component_name }} component for accessibility. You should test your implementation, too.
     </p>
-      <a class="usa-button margin-top-1 font-body-xs" href="{{ site.baseurl }}/components/{{ componentName }}/accessibility-tests">
-        Use {{ componentName }} accessibility tests
+      <a class="usa-button margin-top-1 font-body-xs" href="{{ site.baseurl }}/components/{{ componentKey }}/accessibility-tests">
```

Comment:
Should we use `component_key` here or is that work captured elsewhere?

Reply:
I've added this as a question to answer in #2629. I'll try to fit this work in this sprint because it does require some extra thinking to make sure we are referencing the correct variable.